src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Select.java

```diff
@@ -22,58 +22,61 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static com.amazon.opendistroforelasticsearch.sql.domain.Field.STAR;
+
 /**
  * 将sql语句转换为select 对象
  * 
  * @author ansj
  */
 public class Select extends Query {
 
-	// Using this functions, will cause query to execute as aggregation.
-	private final List<String> aggsFunctions = Arrays.asList("SUM", "MAX", "MIN", "AVG", "TOPHITS", "COUNT", "STATS","EXTENDED_STATS","PERCENTILES","SCRIPTED_METRIC");
+    // Using this functions, will cause query to execute as aggregation.
+    private final List<String> aggsFunctions = Arrays.asList("SUM", "MAX", "MIN", "AVG", "TOPHITS", "COUNT", "STATS","EXTENDED_STATS","PERCENTILES","SCRIPTED_METRIC");
```

Comment:
formatting: missing spaces between parameters in second line.

consider renaming to aggregateFunctions

Reply:
I found this is supposed to be a constant of Set. Fixed it.